substrate/frame/revive/src/evm/runtime.rs

```diff
@@ -48,7 +48,7 @@ type CallOf<T> = <T as frame_system::Config>::RuntimeCall;
 /// We use a fixed value for the gas price.
 /// This let us calculate the gas estimate for a transaction with the formula:
 /// `estimate_gas = substrate_fee / gas_price`.
-pub const GAS_PRICE: u32 = 1_000u32;
+pub const GAS_PRICE: u32 = 1u32;
```

Comment:
Yeah I think either value is fine since they are constant

1 is nice since we can compute  tx.gas = dry_run.fee  / GAS_PRICE, there are no rest to the division